custom_otello_linter/abstract_checkers/scenario_helper.py

```diff
@@ -13,3 +13,33 @@ def get_all_steps(self, class_node: ast.ClassDef) -> List:
                 or isinstance(element, ast.AsyncFunctionDef)
             )
         ]
+
+    def get_allure_decorator(self, scenario_node: ast.ClassDef) -> Union[ast.Call, None]:
```

Comment:
```suggestion
    def get_allure_labels_decorator(self, scenario_node: ast.ClassDef) -> Union[ast.Call, None]:
```
У самого аллюра на самом деле много декораторов. Мы их не используем, конечно, но в будущем путаница может возникнуть потенциально.